### DISCUSSION TITLE
Newpipe like subscriptions

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#160] [Question]
I like the way newpipe deals with subscriptions - without a google account. Maybe even have it be compatible with newpipes subscriptions

**Comment by ghost** [comment]
y

**Comment by ghost** [comment]
> y

Because I am trying to degoogle and not enough of the youtubers I watch are on peertube or odysee. Newpipe is great but I can't get it to play videos at 1080p60 without it buffering every 10 seconds

**Comment by ghost** [comment]
> > y
> 
> Because I am trying to degoogle and not enough of the youtubers I watch are on peertube or odysee. Newpipe is great but I can't get it to play videos at 1080p60 without it buffering every 10 seconds

use newpipe sponserblock then lol

**Comment by ghost** [comment]
> > > y
> > 
> > Because I am trying to degoogle and not enough of the youtubers I watch are on peertube or odysee. Newpipe is great but I can't get it to play videos at 1080p60 without it buffering every 10 seconds
> 
> use newpipe sponserblock then lol

I literally just said it that it keeps buffering so it's really annoying to use.

**Comment by TheJeterLP** [comment]
> > > > y
> > > 
> > > Because I am trying to degoogle and not enough of the youtubers I watch are on peertube or odysee. Newpipe is great but I can't get it to play videos at 1080p60 without it buffering every 10 seconds
> > 
> > use newpipe sponserblock then lol
> 
> I literally just said it that it keeps buffering so it's really annoying to use. 

You should report that go new pipe then.
